sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -115,7 +115,7 @@ DetectedLanguage language = result.PrimaryLanguage;
 Console.WriteLine($"Detected language {language.Name} with confidence {language.Score:0.00}.");
 ```
 
-Please refer to the service documentation for a conceptual discussion of [language detection][langugage_detection].
+Please refer to the service documentation for a conceptual discussion of [language detection][language_dectection].
```

Comment:
Oooh, it's actually spelled incorrectly in the links below.  Would it be possible to correct it there?

Reply:
:| sure thing!